src/api/app/assets/javascripts/webui2/package-view_file.js

```diff
@@ -0,0 +1,29 @@
+var DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+var MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+
+function addChangesEntryTemplate() { // jshint ignore:line
+  var date = new Date(),
+      day = date.getUTCDate().toString(),
+      hours,
+      minutes,
+      seconds,
+      template;
+
+  if (day.length === 1) { day = " " + day; } // Pad single character day value
+  hours = date.getUTCHours();
+  if (hours < 10) { hours = '0' + hours; }
+  minutes = date.getUTCMinutes();
+  if (minutes < 10) { minutes = '0' + minutes; }
+  seconds = date.getUTCSeconds();
+  if (seconds < 10) { seconds = '0' + seconds; }
+
+  template = "-------------------------------------------------------------------\n" +
```

Comment:
I'm not sure if we need to keep the exact same format as the old UI:
```
-------------------------------------------------------------------
Wed Sep 26 18:47:41 UTC 2018 - OBS Instance Superuser <root@localhost>

- 
```

If not, I would go with [Date.toUTCString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toUTCString), which would reduce the amount of code in this function and produce this output:

```
-------------------------------------------------------------------
Wed, 26 Sep 2018 19:05:46 GMT - OBS Instance Superuser <root@localhost>

- 
```

It can be slightly adapted by doing some shenanigans... if it's really needed for some reason. I don't know if some tools are processing this for example...

Reply:
It should be the same as ``osc vc`` does produce:

```
-------------------------------------------------------------------
Thu Sep 27 08:11:23 UTC 2018 - mail/name

```

Reply:
I won't change this in the PR, so marking the conversation as resolved.